syncer/status.go

```diff
@@ -22,27 +22,50 @@ import (
 
 	"github.com/pingcap/dm/dm/config"
 	"github.com/pingcap/dm/dm/pb"
+	"github.com/pingcap/dm/pkg/binlog"
+	"github.com/pingcap/dm/pkg/log"
+	"github.com/pingcap/dm/syncer/metrics"
 )
 
 // Status implements Unit.Status
 // it returns status, but does not calc status.
-func (s *Syncer) Status() interface{} {
+func (s *Syncer) Status(sourceStatus *binlog.SourceStatus) interface{} {
 	total := s.count.Load()
 	totalTps := s.totalTps.Load()
 	tps := s.tps.Load()
+	lag := s.secondsBehindMaster.Load()
 
 	syncerLocation := s.checkpoint.FlushedGlobalPoint()
 	st := &pb.SyncStatus{
-		TotalEvents:  total,
-		TotalTps:     totalTps,
-		RecentTps:    tps,
-		SyncerBinlog: syncerLocation.Position.String(),
+		TotalEvents:         total,
+		TotalTps:            totalTps,
+		RecentTps:           tps,
+		SyncerBinlog:        syncerLocation.Position.String(),
+		SecondsBehindMaster: lag,
```

Comment:
It seems that `lag` is only used here
```suggestion
		SecondsBehindMaster: s.secondsBehindMaster.Load(),
```